.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: ""
+labels: []
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting this issue.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running? Please provide a tag or commit hash.
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description of the issue
+      description: Please explain in details the issue observed.
+      placeholder:
+      value:
```

Comment:
```suggestion
```
I think these could be removed if we're not putting anything in them (since the version field doesn't have them)